acai-ios/CustomViews/InputView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  InputView.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 4/10/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class InputView: UIView {
+    
+    // MARK: View vars
+    var label: UILabel!
+    var line: UIView!
+    var textField: UITextField!
+    
+    // MARK: Constraint Constants
+    private enum FileConstants {
+        static let lineHeight = 1
+    }
+    
+    enum InputType {
+        case hide
+        case show
+    }
+    
+    init(frame: CGRect, title: String, placeholder: String, inputType: InputType, padding: CGFloat) {
+        super.init(frame: frame)
+        backgroundColor = .clear
+        
+        label = UILabel()
+        label.textColor = .acaiBlack
+        label.font = UIFont.avenirNextDemi.withSize(17)
+        label.text = title
+        self.addSubview(label)
+       
+        textField = UITextField()
+        textField.font = UIFont.avenirNextRegular.withSize(14)
+        textField.textColor = .paleOrange
+        let attr: [NSAttributedString.Key: Any] = [
+            .font: UIFont.avenirNextRegular.withSize(14),
+            .foregroundColor: UIColor.placeholderGray
+        ]
+        textField.attributedPlaceholder = NSAttributedString(string: placeholder, attributes: attr)
+        if inputType == .hide {
+            textField.isSecureTextEntry = true
+        }
```

Comment:
can make this `textField.isSecureTextEntry = inputType == .hide`

Reply:
changed this to case statements